jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -127,6 +129,12 @@ function(params) {
                     '--logs.tail.endpoint=' + api.config.logs.tailEndpoint,
                     '--logs.write.endpoint=' + api.config.logs.writeEndpoint,
                   ] else []
+              ) + (
+                if api.config.traces != {} then
+                  [
+                    '--traces.write.endpoint=' + api.config.traces.writeEndpoint,
```

Comment:
Are we missing a `--traces.read.endpoint` too?

Reply:
Ah - there's not a flag for that in the API [yet](https://github.com/observatorium/api/blob/b3a191802b501fb6051e73f2f986240e1e406be4/main.go#L864-L865)

Reply:
that hasn't been added yet (https://github.com/observatorium/api/pull/199)

@esnible is working on it.